app/assets/javascripts/components/alerts/alerts_handler.jsx

```diff
@@ -5,21 +5,49 @@ import { connect } from 'react-redux';
 
 import AlertsList from './alerts_list.jsx';
 import { fetchAlerts, sortAlerts } from '../../actions/alert_actions';
+import MultiSelectField from '../common/filter.jsx';
+
+const ALERTS = [
+  { label: 'ActiveCourseAlert', value: 'ActiveCourseAlert' },
+  { label: 'ArticlesForDeletionAlert', value: 'ArticlesForDeletionAlert' },
+  { label: 'BlockedEditsAlert', value: 'BlockedEditsAlert' },
+  { label: 'ContinuedCourseActivityAlert', value: 'ContinuedCourseActivityAlert' },
+  { label: 'DeletedUploadsAlert', value: 'DeletedUploadsAlert' },
+  { label: 'DiscretionarySanctionsEditAlert', value: 'DiscretionarySanctionsEditAlert' },
+  { label: 'DYKNominationAlert', value: 'DYKNominationAlert' },
+  { label: 'GANominationAlert', value: 'GANominationAlert' },
+  { label: 'NoEnrolledStudentsAlert', value: 'NoEnrolledStudentsAlert' },
+  { label: 'ProductiveCourseAlert', value: 'ProductiveCourseAlert' },
+  { label: 'UnsubmittedCourseAlert', value: 'UnsubmittedCourseAlert' },
+  { label: 'UntrainedStudentsAlert', value: 'UntrainedStudentsAlert' },
+];
 
 const AlertsHandler = createReactClass({
   displayName: 'AlertsHandler',
 
-
   propTypes: {
     fetchAlerts: PropTypes.func,
     alerts: PropTypes.array
   },
 
+  getInitialState() {
+    return {
+      selected: [
```

Comment:
This is okay, but I think moving the filter state to redux would be better. It doesn't make a significant difference for this feature, since it's an isolate React component where state will be lost with the user navigates away. But once the whole of Campaigns gets migrated to React, having the filters in redux will mean that filter state gets retained when the user navigates away. It also keeps the architecture more consistent with the redux standard, where every significant change in application state is done via a redux action.